sw/host/tests/chip/pwm_smoketest/src/main.rs

```diff
@@ -109,6 +110,10 @@ fn main() -> Result<()> {
     uart.clear_rx_buffer()?;
     transport.pin_strapping("RESET")?.remove()?;
 
+    // Wait until test is running.
+    let uart = transport.uart("console")?;
+    UartConsole::wait_for(&*uart, r"Running [^\r\n]*", opts.timeout)?;
+
```

Comment:
Since the PWM smoketest doesn't do too much between OTTF starting the test and that actual PWM testing logic, this seems like it should sufficiently synchronize the device and the host. An even more precise sync would have the device send a UART message (or toggle a pin, etc) when it's actually ready to start the PWM testing part, but that's probably overkill.

Arguably this could be moved to like 123.